tchannel/sync/client.py

```diff
@@ -20,18 +20,11 @@
 
 from __future__ import absolute_import
 
-from collections import namedtuple
-
-from concurrent.futures import TimeoutError
-from tornado import gen
-
-from tchannel import tornado as async
-from tchannel.tornado.hyperbahn import FIRST_ADVERTISE_TIME
-from tchannel.tornado.hyperbahn import AdvertiseError
+from tchannel import TChannel
 from threadloop import ThreadLoop
 
 
-class TChannelSyncClient(object):
+class TChannelSyncClient(TChannel):
```

Comment:
Since you would never have an async client and sync client in the same application, I think it makes more sense to name this `tchannel.sync.TChannel`. The client code would look like so:

```
from tchannel.sync import TChannel

tchannel = TChannel(...)
```

When we move it out into it's own package, it would just be:

```
from tchannel_sync import TChannel

tchannel = TChannel()
```

From the time you have a tchannel instance on, everything is the same, so the import path is really the only thing that matters.